_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -415,6 +415,35 @@ public static function register_endpoints() {
 				),
 			)
 		) );
+
+		// Get and set API keys.
+		register_rest_route(
+			'jetpack/v4',
+			'/service-api-keys/(?P<service>[a-z\-_]+)',
```

Comment:
When I need to have a hyphen in a regular expression, I put it at the end:
`(?P<service>[a-z_-]+)`

Otherwise, I have to think about whether it's a range operator (from slash to underscore) or an escaped hyphen.

The above isn't advice :) Just a bikeshed about what makes sense to me.